trending_data/src/main/java/com/google/sps/data/TrendingTopic.java

```diff
@@ -0,0 +1,6 @@
+package com.google.sps.data;
+
+public class TrendingTopic {
```

Comment:
Need some comment here (and in TwitterResponse) explaining what each class does, what are the fields, etc.